e2e-cypress/files/metadata.yml

```diff
@@ -0,0 +1,6 @@
+---
+metadata:
+  name: Cypress
+  description: "Cypress is a JavaScript-based end-to-end testing framework that doesn't use Selenium at all. It is built on top of Mocha, which is again a feature-rich JavaScript test framework running on and in the browser, making asynchronous testing simple and fun. Technologies: Typescript"
```

Comment:
@georgfedermann the offered technology here is Cypress, not TypeScript. Please consider putting that as a first item under Technologies. Cypress version used is 3.